tests/chicago_participatory_urbanism/test_geocoding_api.py

```diff
@@ -0,0 +1,32 @@
+# add path to use local module files 
+import sys
+import os
+FILE_DIR = os.path.dirname(os.path.abspath(__file__))
+PROJECT_DIR = os.path.dirname(os.path.dirname(FILE_DIR))
+sys.path.append(PROJECT_DIR)
+
+import unittest
+from src.chicago_participatory_urbanism.geocoder_api import GeoCoderAPI
+from src.chicago_participatory_urbanism.ward_spending.location_geocoding import LocationGeocoder
+from cases import address_tests
+
+class TestGeocoderAPI(unittest.TestCase):
```

Comment:
I'll migrate this to PyTest before I merge https://github.com/ward-wise/data-analysis/pull/35